data/wanderer/wanderers start.txt

```diff
@@ -2867,6 +2867,6 @@ mission "Wanderers Ap'arak 3"
 	on complete
 		event "wanderers: the eye opens"
 		event "wanderers: derecho mass production" 16
-		event "wanderers: spera anatrusk colony" 40
+		event "wanderers: spera anatrusk colony" 90
```

Comment:
I'm curious as to the justification for this. I'm guessing it is increasing it to maintain the down time and taking these missions into account. If this is the case, I'm not sure it's necessary. I'm not convinced that the player has to have that much downtime; and I like the idea that these missions are giving the player something to do to whittle down that "sit around and wait" counter. Adding an additional 50 days to it pretty much guarantees that, even with these missions, the player is *still* going to end up having to burn time for a while. (and potentially get lost, or forget where to pick up the next mission, etc).

So if this is necessary, then so be it. But if it's just increasing the counter to maintain a large block of player idle time, then I'd say scrap this change (or reduce it so that the player isn't actually faced with such a vast amount of empty time. There's been more than a few people who have thought this was the end of the story because, well, having to just sit around and waste over a month of time just doesn't make any sense in terms of story pacing. (Yes, makes lots of sense in terms of realism and how long it takes to build stuff)

Reply:
The original time frame didn't leave enough time for the missions to complete before the colony event expired, so you had a situation were the colony opened while you were still in the process of setting up the outfitter. That description included mass activity of freighters coming in and out which seemed at odds with the work the player was doing. I don't have an opinion on how long it should though I'd say ideally it would shortly after the outfitter finished. Alternatively, we could remove the event from the eye opening and trigger it at the end of these missions.

Reply:
Alright, that makes sense, then.

Reply:
I ran through a test and the missions take about 60 days to complete. The current timing leaves 30 days for exploring Korath space or doing additional trading as you are doing the outfitter missions. So the dead zone is 10 days shorter in this version, but longer if you don't pick up the mission.

I also remembered that the other issue with the colony opening is that you get 3 exploration missions dropped on you which nag you every time you land.